crates/store/src/db/sql.rs

```diff
@@ -424,7 +424,8 @@ pub fn select_notes(conn: &mut Connection) -> Result<Vec<NoteRecord>> {
             tag,
             aux,
             merkle_path,
-            details
+            details,
+            execution_hint
```

Comment:
Similar to the other comment, here, and in other places in this file, I'd probably move `execution_hint` to be right after the `aux` field.